config/overlays/e2e/kustomization.yaml

```diff
@@ -24,3 +23,20 @@ patches:
 images:
 - name: quay.io/metal3-io/baremetal-operator
   newTag: e2e
+
+# We cannot use suffix hashes since the kustomizations we build on
+# cannot be aware of what suffixes we add.
+generatorOptions:
+  disableNameSuffixHash: true
+
+# NOTE: These credentials are only meant for testing!
+# DO NOT USE IN PRODUCTION
```

Comment:
hummm, wondering how many would read this 😄

Reply:
Well it should be already obvious that the overlay is for e2e :smile: 
But on a serious note, as discussed with @tuminoid above, I will most likely change this so that we generate the credentials on the fly instead of putting them here.

Reply:
Yeah saw that after I commented , I agree with @tuminoid